### PR TITLE
Fixes #26507 - Don't focus on autocomplete when editing textarea

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
@Laviro can you please take a look?

File: webpack/assets/javascripts/react_app/components/AutoComplete/AutoComplete.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class AutoComplete extends React.Component {
     const { useKeyShortcuts, handleSearch } = this.props;
     const instance = this._typeahead.current.getInstance();
     const { ENTER, FWD_SLASH, BACK_SLASH } = KEYCODES;
-    const didEventCameFromInput = e.target.tagName === 'INPUT';
+    const didEventCameFromInput = e.target.tagName === 'INPUT' || e.target.tagName === 'TEXTAREA';
```
Comment (Ron-Lavi) [inline webpack/assets/javascripts/react_app/components/AutoComplete/AutoComplete.js]: thanks @tbrisker, missed that one !
now that we have two `tagNames`, I suggest destructing it like:
```
const { tagName } = e.target;
const didEventCameFromInput = tagName === 'INPUT' || tagName === 'TEXTAREA';
```

Just looks cleaner, thanks !

Comment (tbrisker) [reply webpack/assets/javascripts/react_app/components/AutoComplete/AutoComplete.js]: Thanks, updated

**Review by Ron-Lavi — APPROVED** [review]
LGTM, Thanks @tbrisker :hugs:

**Review by sharvit — APPROVED** [review]

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @tbrisker.

**Pull request merged**